modal/functions.py

```diff
@@ -471,7 +478,7 @@ def web_url(self) -> str:
     def is_generator(self) -> bool:
         return self._is_generator
 
-    async def _map(self, input_stream: AsyncIterable, order_outputs: bool, kwargs={}):
+    async def _map(self, input_stream: AsyncIterable, order_outputs: bool, return_exceptions: bool, kwargs={}):
```

Comment:
Let's also add this for `starmap` (should be at parity with the `map` interface)